src/Hl7.Fhir.Support.Poco.Tests/TestData/fp-test-patient-errors-recovered.json

```diff
@@ -1,16 +1,87 @@
 {
   "resourceType": "Patient",
```

Comment:
We should manually review the new output (recovered + dumped errors) to verify this after pulling the PR.